torch/serialization.py

```diff
@@ -397,10 +437,27 @@ def default_restore_location(storage, location):
 
 
 def normalize_storage_type(storage_type):
+    """Normalize a given storage type by retrieving the corresponding attribute from the `torch` module.
+
+    Args:
+        storage_type (type): The storage type to be normalized.
+    """
     return getattr(torch, storage_type.__name__)
 
 
 def storage_to_tensor_type(storage):
+    """Convert a given storage type to its corresponding tensor type.
```

Comment:
Similarly, I think it is okay to omit this docstring from this PR since it was not one of the pydocstyle errors in the issue